apps/total-typescript/prettier.config.js

```diff
@@ -0,0 +1,10 @@
+module.exports = {
+  bracketSpacing: false,
+  jsxBracketSameLine: false,
+  printWidth: 80,
+  semi: false,
+  singleQuote: true,
+  tabWidth: 2,
+  trailingComma: 'all',
+  useTabs: false,
+}
```

Comment:
all of our projects in this repo stash this in package.json, if we prefer it here we need to update all of them and the template

Reply:
leftover debris from the original version. got it moved to package.json. 👍